.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## What is the change?
+Remove this line and add a description
+
+## Related issue?
+closes: #issue_number
+
+## How was it tested?
+Remove this line and tell us how the code was tested.
+
+## Checklist:
+Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:
+
+-   [ ] Have you followed the [Contribution Guidelines]()while contributing.
+-   [ ] Have you checked there aren't other open [Pull Requests](https://github.com/BugBustersCommunity/website/pulls) for the same update/change?
+-   [ ] Have you made corresponding changes to the documentation?
+-   [ ] Have you tested the code before submission?
+-   [ ] Have you formatted your code ? (You can use any html,css beautifier)
```

Comment:
@afreenabegum , don't make it interrogative
Instead use simple sentences
change all `Have you` type question to `I have...`
eg -> Have you followed the guidelines? to` I have followed the guidelines.`

Reply:
> @afreenabegum , don't make it interrogative Instead use simple sentences change all `Have you` type question to `I have...` eg -> Have you followed the guidelines? to` I have followed the guidelines.`

done changes, Please do review.